### ISSUE TITLE
whie low accuracy with the restored checkpoints, and smaller dataset?

### ISSUE THREAD

**Issue by cxm01** [dennybritz/cnn-text-classification-tf#103]
I tried this project , it behaves good while training and testing on my own dataset.but I met a strange problem. it has a very low accuracy on smaller dataset, higher accuracy on larger dataset after restoring from the saved checkppoints . I really do not know how to fix this? dose anyone has any advice about this. 
below is the code for classifying, I just comment out the line for training from train.py

 ```python
           # saver.restore
            checkpoint_directory = os.path.join(run_path, 'checkpoints')
            checkpoint_file=tf.train.latest_checkpoint(checkpoint_directory)
            saver.restore(sess, checkpoint_file)

            for batch in batches:
                x_batch, y_batch = zip(*batch)
                #train_step(x_batch, y_batch)
                current_step = tf.train.global_step(sess, global_step)
                if current_step % FLAGS.evaluate_every == 0:
                    print("\nEvaluation:")
                    dev_step(x_dev, y_dev, writer=dev_summary_writer)
                    print("end dev step")
                #if 1: #current_step % FLAGS.checkpoint_every == 0:
                    #path = saver.save(sess, checkpoint_prefix, global_step=current_step)
                    
                    #print("Saved model checkpoint to {}\n".format(path))
                #if current_step == 10:
                    #path = saver.save(sess, checkpoint_prefix, global_step=current_step)
                    #print("Saved and restore model checkpoint to {}............................\n".format(path))
                    #saver.restore(sess, path)
            print("\nlast Evaluation:")
            dev_step(x_dev, y_dev, writer=dev_summary_writer)
            print('end dev step')
            #path = saver.save(sess, checkpoint_prefix, global_step=current_step)
            #print("Saved model checkpoint to {}\n".format(path))
```
